app/components/GameButtonsPanel/GameButtonsPanel.js

```diff
@@ -48,9 +44,7 @@ export default function GameButtonsPanel(props) {
 
   return (
     <div className="buttonsPanel">
-      <ButtonGroup
-        buttons={ buttons }
-      />
+      <ButtonGroup { ...{ buttons, } } />
```

Comment:
A group implies plural buttons, so the name should reflect that: 'ButtonsGroup'.

Reply:
done